apps/expo/src/app/calendar/index.tsx

```diff
@@ -153,99 +182,108 @@ const Calendar = () => {
     <SafeAreaView className="bg-p-100 flex-1 items-center px-4">
       <Stack.Screen options={{ title: "Calendar", headerShown: false }} />
 
-      {/* logo */}
-      <Image source={Logo} className="-mt-10 w-2/5" resizeMode="contain" />
-      <View className="-mt-10 w-full">
-        <Text className="text-p-0 font-headline-sm">
-          Hello {user.firstName}!
-        </Text>
-      </View>
-      <View className="mx-auto mt-5 h-auto w-full">
-        <TabNav currentTab={tabState}>
-          <TabNav.Tab
-            onPress={() => {
-              setTabState(1);
-            }}
-          >
-            <Text className="text-p-0 font-title-md">Week</Text>
-          </TabNav.Tab>
-          <TabNav.Tab
-            onPress={() => {
-              setTabState(2);
-            }}
-          >
-            <Text className="font-title-md">Day</Text>
-          </TabNav.Tab>
-        </TabNav>
-      </View>
-
-      {tabState === 1 && ( // Week
-        <ScrollView className="mt-5 w-full">
-          {sortedDays.length === 0 ? (
-            <Text className="text-p-0 font-title-md mt-5">
-              No activities this week
-            </Text>
-          ) : (
-            sortedDays.map((day, index) => (
-              <View key={index} className="mt-5 w-full">
-                <Text className="text-p-0 font-title-md">
-                  {formatDate(day)}
-                </Text>
-                {groupedWeeklySchedule[day]?.map((activity, idx) => (
-                  <View key={idx} className="mt-2 w-full flex-row">
-                    <View className="w-1/4">
-                      <Text className="text-p-0 font-body-sm mr-2 mt-1">
-                        {getTimeRange(
-                          activity.startTime,
-                          activity.durationMinutes,
-                        )}
-                      </Text>
-                    </View>
-                    <View className="w-3/4 rounded-lg">
-                      <EventTab
-                        activity={{ name: activity.name }}
-                        attending={true}
-                      />
-                    </View>
-                  </View>
-                ))}
-              </View>
-            ))
-          )}
-          <View className="h-20" />
-        </ScrollView>
-      )}
-
-      {tabState === 2 && ( // Day
```

Comment:
The formatting for day view shrinks in this code, did you change the formatting?

<img width="375" alt="image" src="https://github.com/user-attachments/assets/60bb9e11-fcc5-424a-926b-16dbac8f354d">